nuxt/api/albums.ts

```diff
@@ -0,0 +1,71 @@
+import { NuxtAxiosInstance } from '@nuxtjs/axios';
+import {
+  PaginationOptions,
+  PersistedEntity,
+  TimestampedEntity,
+  PaginatedResponse,
+  createParams,
+  usePagination,
+  useIncludes,
+} from '@/api/common';
+import { Reciter } from '@/api/reciters';
+
+/*
+ * Entity Definitions
+ */
+export interface Album extends PersistedEntity, TimestampedEntity {
+  reciter: Reciter;
```

Comment:
An album doesn't always have a `reciter`. 

It always has a `reciterId: string;`.